documentation/src/main/asciidoc/topical/wildfly/Wildfly.adoc

```diff
@@ -0,0 +1,54 @@
+= Updating Hibernate ORM within WildFly
+
+The http://wildfly.org/[WildFly application server] comes with Hibernate as the default JPA provider out of the box.
+This means that you don't need to package Hibernate ORM with the applications you deploy on WildFly,
+instead the application server will automatically enable Hibernate support if it detects that your application works with JPA/Hibernate.
```

Comment:
"works with JPA/Hibernate" -> "requires JPA" ?

The "works with" seems a bit ambiguous as we can't really tell how well it works and/or how it's integrated. Also we don't do any kind of detection for "native" Hibernate API usage. I'd rather stick to a more vague "Requires JPA" in the introductory section and then point to specifics further down.